cardano-ping/src/Cardano/Network/Ping.hs

```diff
@@ -580,6 +642,9 @@ pingClient stdout stderr PingOpts{pingOptsQuiet, pingOptsJson, pingOptsCount, pi
     eprint :: String -> IO ()
     eprint = traceWith stderr
 
+    stringToLBS :: String -> ByteString
+    stringToLBS = TL.encodeUtf8 . TL.pack
```

Comment:
Should we use the encoding defined by the user in his locale? This could make a difference for encodings which are not compatible with `ASCII`.

Reply:
As discussed on Slack, changed to avoid `ByteString` altogether :+1: